pages/implementation/avo-inspector-sdk-reference.mdx

```diff
@@ -47,6 +47,25 @@ In order to ensure our SDK doesn't have a large impact on performance or battery
 In development batching is disabled by default.
 You can change the values using `AvoInspector.setBatchSize(newBatchSize: Int)` and `AvoInspector.setBatchFlustSeconds(newBatchFlushSeconds: Int)` class methods to fine tune the library performance.
 
+## <a name="versions"></a> App versions
+
+When initializing the Inspector SDK you'll be prompted to provide the current app version. Providing correct app version is not required, but doing so significantly improves the Inspector issue reporting. By providing app version the Inspector is able to tell in what version an issue was first seen, and in what version the issue was fixed. Versions also allow the Inspector to report more accurately on the implementation status of your events, by focusing on the implementation status in your latest app version for every source.
+
+We recommend using semantic versions **("1.0.0")** which get incremented on every release, but any type of incremental string or number will do (e.g. build number).
```

Comment:
A bit confusing here that numbers are in will do area and later are in great area, should we move numbers to recommended alongside semantic versioning?

Reply:
Yes that's a good point, we don't care if it's semantic or not, as long as it increments. Updated in c55d6549cf515dab80b7449aec9f8eb5fa8b87dc!